src/lib/haiku-name.ts

```diff
@@ -137,5 +137,6 @@ const bigNumber = Math.pow(2, 12);
 
 export const generate = () => {
 	const rnd = Math.floor(Math.random() * bigNumber);
+	// tslint:disable-next-line:no-bitwise
```

Comment:
This is the only case I had to just disable the rule and I don't want to touch the code here, but I do think the rule makes sense in general